tests/integration/modules/test_idem.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Integration tests for the idem execution module
+"""
+
+# Import futures
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Python libs
+import logging
+
+# Import salt libs
+import salt.modules.idem
+import salt.utils.path
+
+# Import salt testing libs
+from tests.support.case import ModuleCase
+from tests.support.unit import skipIf
+
+log = logging.getLogger(__name__)
+
+
+@skipIf(not salt.modules.idem.HAS_POP[0], salt.modules.idem.HAS_POP[1])
```

Comment:
Would this be better as `@skipIf(not all(salt.modules.idem.HAS_POP))`?

Reply:
The second one gives the reason why, it's a string representation of the import error

Reply:
Oh, never mind, I misread that 😂